units/units.json

```diff
@@ -41747,4 +41747,4 @@
         "Moment_Of_Inertia": "0",
         "Mounts": "{ShieldBreaker;;;;;;;;;;;;;;;1;1}"
     }
-]
+]
```

Comment:
The only difference I see in units.json is that it no longer has the blank line at end of file. Is this intentional?

Reply:
I had a whole elaborate answer about a bug I fixed. However, sometime between the previous commit and this one, I made changes to parser.py which caused it to bork the json file.
Which, this commit fixes...

I'll have to consider whether to close the PR or just remove the JSON. Something to do tomorrow.

Reply:
Sounds good. I think there are other valid changes in this PR, so I would leave it open and just remove the JSON file.

Reply:
units.json now has several real changes. I merged units.description into it as well as fixed a few units that were missing attributes (but don't remember which unfortunately.)
I did not intend to reuse this PR. I simply forgot it existed and kept working on it concurrently with the engine code.